bundle/manifests/authorino-operator.clusterserviceversion.yaml

```diff
@@ -314,6 +314,12 @@ spec:
                       - --leader-elect
                     command:
                       - /manager
+                    env:
+                      - name: DEFAULT_AUTHORINO_IMAGE
```

Comment:
`RELATED_IMAGE_AUTHORINO` 

In the operator bundle building pipeline, there is some logic to catch `RELATED_IMAGE_*` env vars to get dependency images and build dependency tree used for some verification steps. I can try to find the docs out there.

Reply:
In the specific case of Authorino, I worry that `RELATED_IMAGE_` could be bit misleading, because of the override in the `Authorino` CR.

Not that changing would be a big deal for me. I just didn't want to convey that it means the same thing as it does in Limitador.

Reply:
> 1. Authorino CR -> image name
> 2. CSV -> env var RELATED_IMAGE_AUTHORINO of the container (also useful for dev/testing)

Exactly how you described, but notice that Limitador is not the same, right? (Or maybe it is and I missed that in the Limitador CR.)

Last time I checked, in Authorino Operator, the related Authorino image is a default value that can be overridden in the Authorino CR. Whereas in Limitador Operator, the related Limitador image is **NOT** a default value that can be overridden anywhere; rather, if set, the value of the env var is final.

IOW, Limitador Operator has a "related operand image"; Authorino Operator has a "_default_ related operand image", if that makes sense.

> 3. Default image hardcoded in some const default variable (usually `latest`)

Another difference between Limitador and Authorino operators here.

Notice that, for Authorino Operator, we did NOT hard code a default image value for the case where the env var is missing. I'd argue that breaking the deployments if the env var is missing is better than hiding a silent potential problem that could happen by bumping all operands to latest.

Reply:
There is one caveat in the limitador CR that I was not approved to implement: from the CR you can only especify the docker tag and the registry is hardcoded. I need to write RFC for that... and in the end, the image set in the limitador CR is not a supported thing, just useful for dev/testing and a way out in case the user wants to use their own image.

Reply:
TIL: one can override the env var in the Limitador CR. Well, at least the image tag part. Thanks, @eguzki!